service-registration/eureka/src/main/java/io/smallrye/stork/serviceregistration/eureka/EurekaServiceRegistrar.java

```diff
@@ -76,11 +76,14 @@ private Uni<Void> registerApplicationInstance(WebClient client, String applicati
                         .put("name", "MyOwn"))
                 .put("leaseInfo", new JsonObject().put("renewalIntervalInSecs", 10000).put("durationInSecs", 10000));
 
-        return client.post(path + "/eureka/apps/" + applicationId)
+        Uni<Void> response = client.post(path + "/eureka/apps/" + applicationId)
                 .putHeader("content-type", "application/json")
                 .putHeader("accept", "application/json")
                 .sendJson(instance).replaceWithVoid();
 
+        log.info("Instance registered for service {}: {}", applicationId, registration);
```

Comment:
Should you add this log when the Uni is avec running? 

Before the .replaceWithVoid, use .invoke(() -> log.info....)

Reply:
yes!